composer.json

```diff
@@ -150,6 +149,8 @@
         "twig/string-extra": "^3.4",
         "twig/twig": "^3.4.3",
         "wallabag/phpepub": "^4.0.10",
+        "wallabag/rulerz": "dev-master",
+        "wallabag/rulerz-bundle": "dev-master",
```

Comment:
I think it's OK to use dev-master for those, no plan to do a release as we will not develop new features etc, just maintaining support for recent Symfony versions